### DISCUSSION TITLE
Page not found: 404 not displaying after deployment

### DISCUSSION THREAD

**Discussion by SudhaM-Hexaware** [facebook/docusaurus#8099] [Q&A]
Hi, I have **deployed** my docusaurus site in **Azure**. And, whenever the URLs are wrong or broken it is showing 404 error from azure side. But, I wanted to display our **NotFound** page that I have swizzled. 

**Note**: Locally, its working fine!

Any inputs where I went wrong? Or what to change? So that the NotFound page will be displayed after deployment too.

**Comment by juffalow** [comment]
Hi! I don't have experience with **Azure**, but in **AWS** you have *Error pages* and you are able to setup *custom response*. You probably need to do the same in Azure.

<img width="801" alt="Screenshot 2022-09-16 at 10 22 18" src="https://user-images.githubusercontent.com/8142965/190592226-5a6f9f12-d573-4757-8ebf-0a26dce37e24.png">

**Comment by slorber** [comment]
Hey.

I don't know much about Azure for deploying static sites, but we emit a `404.html` file, and it's the responsibility of you or your host to serve it. Most do it by convention, or provide configuration options to make it possible.

@johnnyreilly as you have Azure experience maybe you can help?
